core/src/test/java/org/hibernate/ogm/backendtck/storedprocedures/PositionalParametersStoredProcedureCallTest.java

```diff
@@ -151,8 +151,14 @@ public void testResultSetStaticCallRaw() throws Exception {
 
 			List<?> listResult = storedProcedureQuery.getResultList();
 			assertThat( listResult ).hasSize( 2 );
-			assertThat( ( (Number) listResult.get( 0 ) ).intValue() ).isEqualTo( 2 );
-			assertThat( listResult.get( 1 ) ).isEqualTo( "title'2" );
+			if ( listResult.get( 0 ) instanceof Integer ) {
+				assertThat( ( (Number) listResult.get( 0 ) ).intValue() ).isEqualTo( 2 );
+				assertThat( listResult.get( 1 ) ).isEqualTo( "title'2" );
+			}
+			else {
+				assertThat( listResult.get( 0 ) ).isEqualTo( "title'2" );
+				assertThat( ( (Number) listResult.get( 1 ) ).intValue() ).isEqualTo( 2 );
+			}
```

Comment:
It's been a while, but I think the reason I'm checking if it's a number is because not all databases will return an Integer. If you know that it returns an integer, you can do:
```
assertThat(listResult).contains("title'2");
assertThat(listResult).contains(2);
```

alternatively, if it's not important if it's an Integer or another Number type,
with the latest version of the assertions, you can do something like this:
```
		assertThat( objects ).contains( "Test 2" ) ;

		List<Integer> found = new ArrayList<>();
		objects.forEach( obj -> {
			if ( obj instanceof Number ) {
				found.add( ( (Number) obj ).intValue() );
			}
		} );
		assertThat( found ).containsExactly( 2 );
```

Reply:
I think that in this case we also expect the id to be first, because we have created the stored procedure.

Reply:
I have updated the PR with the changes that you suggested.